src/components/MarkdownProvider/components/Configuration.tsx

```diff
@@ -76,7 +77,8 @@ export const Configuration: React.FC<{ reactPackage: IPackage; components: IComp
       <StyledListItem>
         <StyledListItemLabel isBold>Deps</StyledListItemLabel>
         <StyledMono>
-          npm install react react-dom prop-types styled-components @zendeskgarden/react-theming
+          npm install react react-dom prop-types styled-components
+          {!isTheming && ' @zendeskgarden/react-theming'}
```

Comment:
Rather than a separate prop, could we use `reactPackage.packageName === '@zendeskgarden/react-theming'` to dynamically apply this value?